sympy/printing/tests/test_numpy.py

```diff
@@ -210,49 +214,49 @@ def test_expm1():
         skip("NumPy not installed")
 
     f = lambdify((a,), expm1(a), 'numpy')
-    assert abs(f(1e-10) - 1e-10 - 5e-21) < 1e-22
+    assert abs(f(1e-10) - 1e-10 - 5e-21) <= NUMPY_DEFAULT_EPSILON
 
 
 def test_log1p():
     if not np:
         skip("NumPy not installed")
 
     f = lambdify((a,), log1p(a), 'numpy')
-    assert abs(f(1e-99) - 1e-99) < 1e-100
+    assert abs(f(1e-99) - 1e-99) <= NUMPY_DEFAULT_SMALLEST_NORMAL
```

Comment:
this should probably be based on 1e-99 (e.g. epsilon*1e-99) or we should check that the difference is exactly zero.

Reply:
I have updated this and `test_expm1` such that the epsilon is correctly scaled